provisioning/transport/amqp/src/AmqpTransportHandler.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Sasl;
+using Microsoft.Azure.Amqp.Transport;
+using Microsoft.Azure.Devices.Provisioning.Client.Transport.Http.Models;
+using Microsoft.Azure.Devices.Shared;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.Transport.Amqp
+{
+    internal static class AmqpTransportHandler
+    {
+        public const string clientVersion = "test-client-version";
```

Comment:
TODO: let's have this in common - I think the HTTP one exposes at least the apiVersion.